classes/options-general.php

```diff
@@ -415,24 +415,24 @@ function print_page() {
 														//TODO What the heck is this?? -Michael
 														if ( true ) {
 															$disabled_api = false;
-															$api_how_to   = __( 'Type in an address, state, or zip to geocode the default location.', 'simplemap' );
+															$api_how_to   = __( 'Type in an address, state, or zip to geocode the default location.', 'SimpleMap' );
 														} else {
 															$disabled_api = true;
-															$api_how_to   = __( 'After you enter an API Key, you can type in an address, state, or zip here to geocode the default location.', 'simplemap' );
+															$api_how_to   = __( 'After you enter an API Key, you can type in an address, state, or zip here to geocode the default location.', 'SimpleMap' );
 														}
 														?>
 
 														<td width="150"><label
-																for="default_lat"><?php _e( 'Starting Location', 'simplemap' ); ?></label>
+																for="default_lat"><?php _e( 'Starting Location', 'SimpleMap' ); ?></label>
 														</td>
 														<td>
 															<label for="default_lat"
-															       style="display: inline-block; width: 60px;"><?php _e( 'Latitude:', 'simplemap' ); ?> </label>
+															       style="display: inline-block; width: 60px;"><?php _e( 'Latitude:', 'SimpleMap' ); ?> </label>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 160 characters

<!-- Comment by [SideCI](https://sideci.com) -->